packages/crashlytics/lib/handlers.js

```diff
@@ -28,17 +28,22 @@ export function createNativeErrorObj(error, stackFrames, isUnhandledRejection) {
   nativeObj.frames = [];
   for (let i = 0; i < stackFrames.length; i++) {
     const { columnNumber, lineNumber, fileName, functionName, source } = stackFrames[i];
-    const subStrLen = fileName.indexOf('?') < 0 ? fileName.length : fileName.indexOf('?');
-    const fileNameParsed =
-      fileName && fileName.length ? fileName.substring(0, subStrLen) : '<unknown>';
+    let fileNameParsed = '<unknown>';
+    if (fileName) {
+      const subStrLen = fileName.indexOf('?');
+      if (subStrLen < 0) {
+        fileNameParsed = fileName;
+      } else if (subStrLen > 0) {
+        fileName.substring(0, subStrLen);
```

Comment:
is fileName used after this anywhere? Seems like the result of this is not used because fileNameParsed is what is used below?

Reply:
Oh my, you are right, I forgot `fileNameParsed = ` -- just updated the PR.